pkg/execute/executor.go

```diff
@@ -116,19 +116,20 @@ const (
 )
 
 // Execute executes commands and returns output
-func (e *DefaultExecutor) Execute() interactive.Message {
-	// TODO: Pass context from bots to this method
-	ctx := context.Background()
-
-	var (
-		command       = utils.RemoveHyperlink(e.message)
-		clusterName   = e.cfg.Settings.ClusterName
-		inClusterName = utils.GetClusterNameFromKubectlCmd(command)
-		args          = strings.Fields(strings.TrimSpace(command))
-		empty         = interactive.Message{}
-		botName       = e.notifierHandler.BotName()
-	)
+func (e *DefaultExecutor) Execute(ctx context.Context) interactive.Message {
+	empty := interactive.Message{}
+	rawCmd := utils.RemoveAnyHyperlinks(e.message)
+	rawCmd = strings.NewReplacer(`“`, `"`, `”`, `"`, `‘`, `"`, `’`, `"`).Replace(rawCmd)
+	clusterName := e.cfg.Settings.ClusterName
+	inClusterName := utils.GetClusterNameFromKubectlCmd(rawCmd)
+	botName := e.notifierHandler.BotName()
+
+	execFilter, err := extractExecutorFilter(rawCmd)
+	if err != nil {
+		return e.respond(err.Error(), rawCmd, execFilter.FilteredCommand(), botName)
```

Comment:
this one can panic as the `execFilter` is nil in case of error: e.g. `kc get po -A --filter “botkube” --filter “test”`

Reply:
Thx! will investigate and fix.

Reply:
Done.